File: 20190519python/0349. Intersection of Two Arrays.py
class Solution:
    def intersection(self, nums1: List[int], nums2: List[int]) -> List[int]:
        rep=[]
        nums1_set=set(nums1)
        nums2_set=set(nums2)
        for num in nums2_set:
            if num in nums1_set:
                rep.append(num)
        return rep

File: archives/C#/0349. Intersection of Two Arrays.cs
public class Solution {
    public int[] Intersection(int[] nums1, int[] nums2) {
        HashSet<int> nums1Set=new HashSet<int>(nums1);
        HashSet<int> nums2Set=new HashSet<int>(nums2);
        nums1Set.IntersectWith(nums2Set);
        result=nums1Set.ToArray();
        return result;
    }
}
